README.md

```diff
@@ -11,4 +11,15 @@ Pre-release content is encrypted until published and only shared with permitted
 - brian_url   http://localhost:8083/
 - use_beta_publisher   false
 - DATASET_API_URL http://localhost:22000
-- DATASET_API_AUTH_TOKEN FD0108EA-825D-411C-9B1D-41EF7727F465
+
+Creating a directory and setting it as `zebedee_root` will allow content to persist between launches of zebedee, otherwise a new directory will be created every time. 
```

Comment:
🙌 This one line has been needed in the documentation for so long, thank you!

Reply:
I genuinely didn't know this was a thing, after a year here, and only discovered last week. 😹